source/html/renderer/src/host-config.ts

```diff
@@ -69,62 +79,85 @@ function parseColorPaletteDefinition(obj: any): IContainerStyleDefinition {
     } : null;
 }
 
-export interface IAdaptiveCardConfig {
-    allowCustomStyle: boolean
+export class AdaptiveCardConfig {
+    allowCustomStyle: boolean;
 }
 
-function parseAdaptiveCardConfiguration(obj: any): IAdaptiveCardConfig {
+function parseAdaptiveCardConfiguration(obj: any): AdaptiveCardConfig {
```

Comment:
Since AdaptiveCardConfig is now a class, this method should be moved to the class and should be renamed into simply "parse"